recipe/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "fe25519" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fe25519-{{ version }}.tar.gz
```

Comment:
`fe25519` should be replaced with `{{ name }}`

Reply:
Good eye!